src/App.tsx

```diff
@@ -1,27 +1,99 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { Box, Button, Typography } from '@mui/material';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [isActiveButtonLag, setIsActiveButtonLag] = useState(false);
+  const [isActiveButtonL5fg, setIsActiveButtonL5fg] = useState(false);
+  const [isActiveButtonLrg, setIsActiveButtonLrg] = useState(false);
```

Comment:
I think in this way it`s better to create one variables, which can accept three different values

Reply:
good point! thank you! fixed